eip155/caip10.md

```diff
@@ -44,9 +44,9 @@ The `chain_id` string will be ammended as follows:
 
 ```
 chain_id:    namespace + ":" + network_id + ":" + reference
-network_id:    [0-9]{1,19}
 namespace:   [-a-z0-9]{3,8}
-reference:   0x[a-fA-F0-9]{1,32}
+network_id:    [0-9]{1,64}
+reference:   0x[a-fA-F0-9]{1,40}
```

Comment:
It must be 40 chars, not between 1 and 40

```suggestion
reference:   0x[a-fA-F0-9]{40}
```

Reply:
gooooooooooood point

Reply:
amazing how much egg fits on this face of mine